docs/core-concepts/bot-sequences/introduction-to-bot-sequences-and-segments.mdx

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: 'Introduction to Bot Sequences and Segments'
+---
+
+# Introduction to Bot Sequences and Segments
+
+import Placeholder from '../../partials/_placeholder.mdx';
+
+<Placeholder />
```

Comment:
I think it could be nice to have the label: `Getting Started with Bot Sequences and Segments`
This way the label would match up with the first page of each section
<img width="281" alt="image" src="https://github.com/user-attachments/assets/25e1eddb-1beb-4ec9-8ca1-f30259f9fc75">

Reply:
Updated